oeps/oep-0002.rst

```diff
@@ -0,0 +1,99 @@
+==========================
+OEP-2: Repository Metadata
+==========================
+
++---------------+-------------------------------------------+
+| OEP           | :doc:`oep-2`                              |
++---------------+-------------------------------------------+
+| Title         | Repository Metadata                       |
++---------------+-------------------------------------------+
+| Last-Modified | 2016-04-21                                |
++---------------+-------------------------------------------+
+| Author        | Calen Pennington <cale@edx.org>           |
++---------------+-------------------------------------------+
+| Arbiter       |                                           |
++---------------+-------------------------------------------+
+| Status        | Draft                                     |
++---------------+-------------------------------------------+
+| Type          | Best Practice                             |
++---------------+-------------------------------------------+
+| Created       | 2016-04-21                                |
++---------------+-------------------------------------------+
+
+Abstract
+========
+
+Proposes a standard format for repository metadata for Open edX projects.
+
+Specification
+=============
+
+Each repo in the Open edX system will include a file ``openedx.yml``, with the
+following fields:
+
+``owner``: string
+    The github user/team responsible for the repository.
+
+``nick``: string
+    A short-name for the repository used by reporting tools.
+
+``area``: string
+    Used to group repositories by in reporting tools. Sample areas: core,
+    mobile analytics.
+
+``oeps``: dictionary
+    A list of keys corresponding to Best Practice OEPs (in the format
+    ``oep-n``). Each value would be one of ``True``, ``False``, or a
+    reason dictionary. ``True`` and ``False`` would indicate accordance (or
+    lack thereof) with the specified OEP. A reason dictionary
+    must have ``state`` and ``reason`` keys. The ``state`` key is a boolean,
+    and ``reason`` is a string that should explain why it's ok that the
+    ``state`` is ``False`` (this will be displayed in reporting tools).
+
+    If an OEP isn't listed in the ``oeps`` dictionary, then it is assumed to be
+    ``False``, unless the reporting tool can auto-detect accordance.
+
+
+Motivation
+==========
+
+One of the primary goals for Best Practice OEPs is to standardize techniques
+and tools across the Open edX repositories. However, without tooling, it will
```

Comment:
How do you define "Open edX repositories", precisely? Does that include XBlocks, for examples? Would there be categories? I imagine not all OEPs would apply to all types of repos?